plugins/eni/README.md

```diff
@@ -13,7 +13,8 @@ An example configuration for invoking the plugin is listed next:
     "eni":"eni-eni01en1",
     "ipv4-address":"172.31.31.65/20",
     "mac":"01:23:45:67:89:ab",
-    "block-instance-metadata":true
+    "block-instance-metadata":true,
+    "stay-down":true,
```

Comment:
Nit: It might make more sense to have stay-down:false in the example, since that's what most tasks would do.

Reply:
Good point. done.